core/controllers/email_dashboard.py

```diff
@@ -208,6 +209,11 @@ class QueryStatusCheckHandler(
 
     @acl_decorators.can_manage_email_dashboard
     def get(self) -> None:
+        """Retrieves user query with some ID.
+
+        Raises:
+            InvalidInputException. Invalid query id.
```

Comment:
I think we should change the error message here, reason:

Not found is not equivalent to wrong/Invalid query ID.